soc/espressif/Kconfig.sysbuild

```diff
@@ -1,10 +1,144 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+config BOOTLOADER_MCUBOOT_ESPRESSIF
+	bool "MCUboot Espressif Port"
+	default y if BOOTLOADER_CUSTOM
+	depends on SOC_FAMILY_ESPRESSIF_ESP32 && BOOTLOADER_CUSTOM
+	help
+	  Enable MCUboot Espressif Port as the bootloader to use
+
 choice BOOTLOADER
 	default BOOTLOADER_MCUBOOT
 endchoice
 
 choice MCUBOOT_MODE
 	default MCUBOOT_MODE_OVERWRITE_ONLY if SOC_FAMILY_ESPRESSIF_ESP32
 endchoice
+
+if BOOTLOADER_MCUBOOT_ESPRESSIF
+
+config ESP_MCUBOOT_IMGTOOL_SIGN_VERSION
+	string "Version to pass to imgtool when signing"
+	default "$(APP_VERSION_TWEAK_STRING)" if "$(VERSION_MAJOR)" != ""
```

Comment:
this doesn't work, this is an application version, not a sysbuild version

Reply:
Removed